### DISCUSSION TITLE
How to crate this?

### DISCUSSION THREAD

**Discussion by taryelkazimov** [requarks/wiki#3126] [Help / Questions]
Hi, how we can create lines like in this page. 
Docker / Microsoft  etc - when we scroll ower they are like the line in excel but different format with a link.
https://docs.requarks.io/install
![wiki_line](https://user-images.githubusercontent.com/35764132/108811352-8d540400-75c6-11eb-8dac-15f58f5c1b76.jpg)

**Comment by Phil-hostwiki** [comment]
You can view the page source by clicking on Page Actions  >  View Source.
see: https://docs.requarks.io/s/en/install

The output is:
```
## By Platform
- [Docker](/install/docker)
- [Heroku](/install/heroku)
- [Kubernetes](/install/kubernetes)
- [Linux](/install/linux)
- [macOS](/install/macos)
- [Windows](/install/windows)
{.links-list}
```
